.vortex/installer/tests/Functional/Handlers/GitleaksHandlerProcessTest.php

```
@@ -0,0 +1,22 @@
+
+namespace DrevOps\VortexInstaller\Tests\Functional\Handlers;
+
+use DrevOps\VortexInstaller\Prompts\Handlers\Gitleaks;
+use PHPUnit\Framework\Attributes\CoversClass;
+
+#[CoversClass(Gitleaks::class)]
+class GitleaksHandlerProcessTest extends AbstractHandlerProcessTestCase {
+
+  public static function dataProviderHandlerProcess(): \Iterator {
+    yield 'gitleaks_enabled' => [
+      static::cw(fn($test): true => $test->prompts[Gitleaks::id()] = TRUE),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\DrevOps\VortexInstaller\Prompts\Handlers\Gitleaks' in method 'dataProviderHandlerProcess'. (undefined)

(StaticAccess)